File: Problem2/claude1.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    int n;
    scanf("%d", &n);
    vector<long long> a(n);
    long long sum = 0;
    for (int i = 0; i < n; i++){
        scanf("%lld", &a[i]);
        sum += a[i];
    }
    if (sum % n == 0) printf("YES\n");
    else printf("NO\n");
    return 0;
}

File: Problem2/claude1.py
import sys

def main():
    data = sys.stdin.read().split()
    n = int(data[0])
    a = list(map(int, data[1:1+n]))
    total = sum(a)
    print("YES" if total % n == 0 else "NO")

if __name__ == "__main__":
    main()